docs/_includes/migrating-from-asciidoc.adoc

```diff
@@ -0,0 +1,353 @@
+
+|+ifeval::[]+ 
+|Evaluates any Python expression.
+|Evaluates simple logical expressions testing the value of attributes. See <<ifeval-directive>>.
+
+|Block delimiters
+|Delimiters do not have to match in length.
+|The length of start and end delimiters must match exactly.
+
+|Table cell contains Asciidoc
+|`a\|` or `asciidoc\|`.
+|`a\|` only.
+
+// NO - deprecated in adp 8, so dont need to address.
+//|Deprecated tables
+//|Yes
+//|No (you don't want them anyway)
+
+|====
+
+
+[[migrate-deprecated]]
+=== Deleted and Deprecated Attributes
+
+// Based on the previous table, but a lot of things shown as not-implemented actually are.
+These are attributes that either no longer exist, work differently, or have better alternatives.
+
+.Deleted and deprecated attributes
+[cols="<20a,<20a,<60a"]
+|====
+|{adp} |{adr} |Notes
+
+
+|`big`, `small`, `underline`, `overline`, `line-through`, colors
+|_deprecated_
+|Character attributes to apply formatting directly. 
+Usually better to apply a role, then apply the formatting based on that role by using a stylesheet.
+
+// old table says not, but...
+|`halign`, `valign` for table cells
+|Column and cell specifiers
+|See <<cell>>.
+
+|`infile`
+.2+|_not implemented_
+.2+|Provides the name and directory of the current document. 
+(Distinct from `docfile`, because `infile` may be an included document, and `docfile` is always the master document.)
+No {adr} equivalent.
+
+|`indir`
+
+|`asciidoc`
+|`asciidoctor` 
+|{adp} sets `asciidoc` to show that it is the current processor. 
+{adr} sets `asciidoctor` instead.
+
+// Ref migration guide
+|`toc2`
+.2+|`toc`
+.2+|Combined in a single attribute, see <<toc>>.
+
+// Ref migration guide
+|`toc-placement`
+
+// No checked
+|`options="pgwide"`
+|_not implemented_
+|DocBook attribute to make tables full page width, whatever the current indent.
+No {adr} equivalent.
+
+// Ref ap UG. I know it workls in ad tables; no ref to ad paragrphs so assume not implemented.
+|options="unbreakable"
+|
+|In {adr}, this only works for tables, not paragraphs.
+
+// http://www.methods.co.nz/asciidoc/userguide.html#X39
+// an ugly hack so not implemented
+|`plaintext`
+|_not implemented_
+|{adp} uses this to suppress inline substitutions and retain block indents when importing large blocks of plain text.
+{adr} deliberately does not implement it; the closest {adr} equivalent is a pass block.
+
+// old table
+|`replacements2`
+|`post_replacements`
+|Renamed.
+
+// does this need an explanation?
+|`presubs`
+|-
+|Not required.
+
+// rarely used so dont mention it
+//|`sgml`
+//|_not implemented_
+//|SGML is archaic and has been replaced by XML.
+
+// It may be a perversion, but it is a useful perversion!
+|`showcomments`
+|_not implemented_
+|In {adp}, turns single line comments into DocBook "remark" elements.
+{adr} considers this a bad use of comments. 
+If you want to pass remarks to the output, use an extension, or:
+----
+ifdef:showcomments[]
+<remark>Your comment here</remark>
```

Comment:
You'll need to put this in a pass block (or pass macro if inline). Also, since this is for DocBook, the ifdef should filter for that as well. In all, that's:

```
ifdef::showcomments+basebackend-docbook[]
++++
<remark>Your comment here</remark>
++++
endif::[]
```